parcel_test.go

```diff
@@ -54,8 +54,7 @@ func TestAddGetDelete(t *testing.T) {
 	// проверьте, что посылку больше нельзя получить из БД
 	err = store.Delete(id)
 	require.NoError(t, err)
-	//для проверки, что посылку получить нельзя попытался использовать данную строчку но
-	//почему, то пишет ошибку assert.ErrorIs(t, err, sql.ErrNoRows)
+	assert.ErrorIs(t, store.Delete(id), err)
```

Comment:
Так, ты тут сравниваешь ответ от двух вызовов, и было странно если они бы не были равны.

Нужно проверять, что удаление проходит без ошибок, а при получении у нас ошибка, делается это так

```suggestion
require.NoError(t, err)

_, err = store.Get(parcel.Number)
require.ErrorIs(t, err, sql.ErrNoRows)
```